WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit Price/ProductPriceSettingsViewController.swift

```diff
@@ -333,17 +333,57 @@ private extension ProductPriceSettingsViewController {
     func configurePrice(cell: UnitInputTableViewCell) {
         let cellViewModel = Product.createRegularPriceViewModel(regularPrice: viewModel.regularPrice, using: CurrencySettings.shared) { [weak self] value in
             self?.viewModel.handleRegularPriceChange(value)
+            self?.setCellPriceAccessibility(price: cell.value, cell: cell)
         }
         cell.selectionStyle = .none
         cell.configure(viewModel: cellViewModel)
+        setCellPriceAccessibility(price: cell.value, cell: cell)
+    }
+
+    func setCellPriceAccessibility(price: String?, cell: UnitInputTableViewCell) {
+        let value = cell.value.isEmpty == false ? cell.value : NSLocalizedString("empty", comment: "Accessibility text for when a price is empty")
+        let currency = cell.value.isEmpty == false ? CurrencySettings.shared.currencyCode.rawValue : ""
```

Comment:
I think it might be worth separating the strings for empty and non-empty values, so that the empty price uses a separate string `NSLocalizedString("Price empty", ...)`. would like to hear what @shiki thinks here!

Reply:
If we use `accessibilityLabel` and `accessibilityValue` as I described in another comment below then we won't need to worry about having a separate string. 🙂 

value | accessibilityLabel | accessibilityValue
-|--------|-------
  null      |       "Price" | "Empty"
0 | "Price" | "0.00 CAD"
123 | "Price" | "123.00 CAD"